doc/classes/VisualShaderNodeRotationByAxis.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShaderNodeRotationByAxis" inherits="VisualShaderNode" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
```

Comment:
Please fill the documentation.

Reply:
@KoBeWi Should this be the same as the description I wrote inside the source code or should this be more in-depth?

Reply:
It doesn't need to be very detailed, check how other VisualShader nodes are documented. Just be sure to fill all fields.